ssl/quic/quic_channel.c

```diff
@@ -2673,6 +2748,29 @@ static void free_token(const unsigned char *buf, size_t buf_len, void *arg)
     OPENSSL_free((unsigned char *)buf);
 }
 
+/**
+ * ch_restart - Restarts the QUIC channel by simulating loss of the initial
+ * packet. This forces the packet to be regenerated with the updated protocol
+ * version number.
+ *
+ * @ch: Pointer to the QUIC_CHANNEL structure.
+ *
+ * Returns 1 on success, 0 on failure.
+ */
+static int ch_restart(QUIC_CHANNEL *ch)
+{
+
+    /*
+     * Just pretend we lost our first initial packet, so it gets
+     * regenerated, with our updated protocol version number
+     */
+    if (!ossl_ackm_mark_packet_pseudo_lost(ch->ackm, QUIC_PN_SPACE_INITIAL,
```

Comment:
NIT: we can return value which comes from `ossl_ackm_mark_packet_pseudo_last()`, no need for if.

Reply:
fixed